src/shared/components/ChannelCover/ChannelCover.tsx

```diff
@@ -53,12 +53,13 @@ const ChannelCover: React.FC<ChannelCoverProps> = ({
             </EditCoverMobileButton>
             {coverPhotoUrl && (
               <>
-                <RemoveCoverDesktopButton onClick={onCoverRemoveClick} icon={<SvgGlyphTrash />} variant="tertiary">
+                {/* TODO: Uncomment as soon as asset removal feature becomes available */}
+                {/* <RemoveCoverDesktopButton onClick={onCoverRemoveClick} icon={<SvgGlyphTrash />} variant="tertiary">
```

Comment:
We probably should comment the imports and styled component definition as well, as to not have dangling imports